core/kernel/classes/class.Class.php

```diff
@@ -228,6 +228,52 @@ public function createSubClass($label = '', $comment = '', $uri = "")
         return $returnValue;
     }
 
+    /**
+     * Retrieves a direct subclass by label.
+     *
+     * @param  string label
+     * @return core_kernel_classes_Class|null
+     */
+    public function retrieveSubClassByLabel($label)
```

Comment:
can we set type hints and a return type here and below?

Reply:
Sure. I don't know the current Tao policy regarding this, and I kept the style of the rest of the file... You tell me. ;-)